schema/dcschema_enum_instances.mcf

```diff
@@ -1376,3 +1371,1599 @@ typeOf: dcs:ICD10Section
 Node: dcid:Overweight
 name: "Overweight"
 typeOf: dcs:HealthBehaviorEnum
+
+Node: dcid:BLS_SS47021
```

Comment:
1. Is there a specific name for this set of product enums? Similar to how BLS publishes NAICS which is for industries. If there is, we could replace BLS with the more specific name.
2. Can you actually put this into a separate file? There's enough I think to warrant one.

Will merge this PR soon unless there are some concerns that I am not aware of/forgotten.

@intrepiditee

Reply:
1. BLS just refers to them as items in this file https://download.bls.gov/pub/time.series/cu/cu.item.
2. Will create a new file

Reply:
Ok. Can we move it to a new file and use BLSItem/SS47021 etc? LMK if you want to do it here, or in a follow up PR @intrepiditee

Reply:
done